kotlinx-coroutines-debug/build.gradle

```diff
@@ -21,7 +21,11 @@ configurations {
 dependencies {
     compileOnly "junit:junit:$junit_version"
     shadowDeps "net.bytebuddy:byte-buddy:$byte_buddy_version"
-    shadowDeps "net.bytebuddy:byte-buddy-agent:$byte_buddy_version"
+    compile "net.bytebuddy:byte-buddy-agent:$byte_buddy_version"
+    compile 'io.projectreactor.tools:blockhound:1.0.2.RELEASE'
```

Comment:
please extract BH and JNA versions to properties as well

Reply:
Done.